src/storage/exec/IndexEdgeNode.h

```diff
@@ -66,6 +62,7 @@ class IndexEdgeNode final : public RelNode<T> {
       edges.emplace_back(std::move(edge));
       iter->next();
     }
+    int64_t count = 1;
```

Comment:
Why `count` is initialized with 1?

Reply:
> Why `count` is initialized with 1?

 limit has three types of boundary value:
limit == -1, This means no limits and is a default value. 
limit == 0, This means don't need to return any data. this one don't need to any data scan. handler in LookupProcessor.cpp:26
limit > 0, true used for limit. the count number begin with 1.

Reply:
I mean you should initialize with 0? See line 82 `limit_ > 0 && ++count > limit_`

Reply:
It looks ok
Because here put the data in first, and then judge
If `count` is initialized to 0, the code on line 82 is as follows:
```
if (limit_> 0 && ++count >= limit_) {
    break;
}
```

Reply:
But I prefer to initialize `count` to 0, wdyt?

Reply:
> But I prefer to initialize `count` to 0, wdyt?

Aha, the point of ambiguity is here, they are all right. the code readability is a bit low.
Changed the init count to 0.